docker-compose.yml

```diff
@@ -2,14 +2,18 @@ version: "3.6"
 services:
       
   elasticsearch:
-    image: elasticsearch:5.6.11
+    #image: elasticsearch:5.6.11
+    #note that the docker hub version doesn't allow restores for some reason...
+    image: docker.elastic.co/elasticsearch/elasticsearch:5.6.11
     ports:
       - 9200:9200
     environment:
       # assign more memory to JVM 
-      - ES_JAVA_OPTS=-Xms12g -Xmx12g
-      - xpack.security.enabled=false
-      - bootstrap.memory_lock=true
+      - "ES_JAVA_OPTS=-Xms12g -Xmx12g"
+      - "xpack.security.enabled=false"
+      - "bootstrap.memory_lock=true"
+      #allow downloading from open targets google buckets
+      - repositories.url.allowed_urls=https://storage.googleapis.com/open-targets-data-releases/*
```

Comment:
I think elasticsearch is missing a list instead an element there. This is what is in the elasticsearch configuration that it is infrastructure elasticsearch folder repositories.url.allowed_urls: ["https://s3-eu-west-1.amazonaws.com/*","https://storage.googleapis.com/*"]

Reply:
But we don’t have anything on AWS anymore? I’ll take a look at the rest to see if thweres anything else from there we can add.

Reply:
Fair enough, makes sense for others that might want to do their own backups / restores. I've added AWS URLs now.